update.sh

```diff
@@ -30,9 +30,15 @@ for xsl in svd/devices/*\.xsl; do
   rm -rf src/
   form -i lib.rs -o src
   rm lib.rs
+
+  # remove last characters, because they just represent the memory size
   ${SED} -i "s/${CHIP}/${CHIP:0:9}/g" src/lib.rs
+
   cargo +nightly fmt
   rustfmt +nightly build.rs
 
+  # ignore all clippy warnings/errors in auto-generated code
+  ${SED} -i "s/#\!\[no_std\]/#\!\[allow\(clippy::all\)\]\n#\!\[no_std\]/g" src/lib.rs
```

Comment:
This is needed because previously, I had manually modified the `lib.rs`'s to add `#![allow(clippy::all)]`